models/issue_stopwatch_test.go

```diff
@@ -0,0 +1,51 @@
+package models
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestCancelStopwatch(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
+	err := CancelStopwatch(1, 1)
+	assert.NoError(t, err)
+	AssertNotExistsBean(t, &Stopwatch{UserID: 1, IssueID: 1})
+
+	_ = AssertExistsAndLoadBean(t, &Comment{Type: CommentTypeCancelTracking, PosterID: 1, IssueID: 1})
+
+	assert.Nil(t, CancelStopwatch(1, 2))
+}
+
+func TestStopwatchExists(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
+	assert.True(t, StopwatchExists(1, 1))
+	assert.False(t, StopwatchExists(1, 2))
+}
+
+func TestHasUserAStopwatch(t *testing.T) {
```

Comment:
Rename to `TestHasUserStopwatch` and there are still other places where `HasUserAStopwatch` is used that also needs to be fixed